templates/SilverStripe/SiteConfig/Includes/SiteConfigLeftAndMain_EditForm.ss

```diff
@@ -24,7 +24,7 @@
 			<% end_loop %>
 			<% if $Controller.LinkPreview %>
 				<a href="$Controller.LinkPreview" class="cms-preview-toggle-link ss-ui-button" data-icon="preview">
-					<% _t('SilverStripe\Admin\LeftAndMain.PreviewButton', 'Preview') %> &raquo;
+					<%t SilverStripe\Admin\LeftAndMain.PreviewButton 'Preview' %> &raquo;
```

Comment:
Can you please use `\\` to escape the localisation strings? Sorry if it wasn't clear earlier. I think text collector has some issues unless you strictly escape special chars.